src/app/component/Form/Form.component.js

```diff
@@ -156,7 +156,7 @@ class Form extends Component {
             <form
               block="Form"
               mods={ { isInvalid: !fieldsAreValid } }
-              ref={ this.form }
+              ref={ (ref) => { this.form = ref; } }
```

Comment:
Let's use `ref => this.form = ref` and by the way, why do you do so?

Reply:
Hmm I get `no-return-assign` lint error if I add it as you described. This is necessary to get form reference so `reset()` could be used upon submitting the form to clear form text inputs.